lib/backend/k8s/conversion/conversion.go

```diff
@@ -700,12 +693,9 @@ func (c Converter) SplitProfileRevision(rev string) (nsRev string, saRev string,
 		return
 	}
 
-	if c.AlphaSA == false {
-		nsRev = rev
-		return
-	}
-
 	revs := strings.Split(rev, "/")
+	// TODO(saumoh): Ask @casey: Should this be an error? or should we return nsRev, ""
```

Comment:
I think this is fine, since it's always passed a value as created by `JoinProfileRevisions`, which means it will always have length 2.